exo-go/src/types/docker_config.go

```diff
@@ -4,7 +4,7 @@ package types
 // docker-compose.yml
 type DockerConfig struct {
 	Image         string            `yaml:",omitempty"`
-	Build         string            `yaml:",omitempty"`
+	Build         map[string]string `yaml:",omitempty"`
```

Comment:
Are we calling the Dockerfile inside the test folder something else? Can't we just point `build` to `./service` vs `./service/tests`?